nvme-cli.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: nvme-cli
+  version: 2.10.2
+  epoch: 0
+  description: NVM-Express user space tooling for Linux
+  copyright:
+    - license: GPL-2.0-only
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - json-c
```

Comment:
we don't need json-c unless you specify a build option `-Djson-c=enabled`

Reply:
thanks for the review. Removed `json-c` given `libnvme-dev` already pulls in json-c. The binary is dynamically linking to json-c out of the box.

```bash
55307245e915:/work/packages# ldd $(command -v nvme)
        linux-vdso.so.1 (0x00007ffc9c7d0000)
        libnvme.so.1 => /usr/lib/libnvme.so.1 (0x00007c90d1d4b000)
        libnvme-mi.so.1 => /usr/lib/libnvme-mi.so.1 (0x00007c90d1d41000)
        libc.so.6 => /lib/libc.so.6 (0x00007c90d1b4c000)
        libjson-c.so.5 => /usr/lib/libjson-c.so.5 (0x00007c90d1b3c000)
        libcrypto.so.3 => /usr/lib/libcrypto.so.3 (0x00007c90d1600000)
        libkeyutils.so.1 => /usr/lib/libkeyutils.so.1 (0x00007c90d1b33000)
        /lib64/ld-linux-x86-64.so.2 (0x00007c90d1dfa000)
```